include/aikido/planner/dart/detail/DartPlannerAdapter-impl.hpp

```diff
@@ -0,0 +1,35 @@
+#include "aikido/planner/dart/DartPlannerAdapter.hpp"
+
+using aikido::statespace::dart::MetaSkeletonStateSpace;
```

Comment:
Technically, `-impl.hpp` is also a header file so using `using namespace ~` can pollute the namespaces. Please remove this.